### PR TITLE
🚧 Fix settings not opening

### LINKED ISSUE #5715: Can't go to settings a second time
## 🐛 Bug Report
In iOS you can't go to settings a second time after you exited tapping the timeline button in the tab bar.

## Context
Build/Branch: develop and AppStore version
Phone OS: 12.4
Phone model: Xs

## To reproduce
- Open the app, go to settings
- Tap on the timeline button down on the tab bar to go back to the timeline
- Try to open settings again. Does it work?

### PR DESCRIPTION
**Pull request by sofokli1**
## What's this?
Check the issue description

### Relationships
Closes #5715

## Why do we want this?
settings need to open always

## How is it done?
replace `FromAsync` to `FromAction`

### Why not another way?
can't think of any

### Side effects
only good stuff

## Review hints
run it

## :squid: Permissions
:squid:

### CODE REVIEW

File: Toggl.Core.UI/ViewModels/MainViewModel.cs [hunk 1]
```diff
@@ -394,7 +394,7 @@ private Task startTimeEntry(bool useDefaultMode)
             var initializeInManualMode = useDefaultMode == userPreferences.IsManualModeEnabled;
 
             OnboardingStorage.StartButtonWasTapped();
-         
+
```
[the remark this answers is not in the document]
Comment (sofokli1) [reply Toggl.Core.UI/ViewModels/MainViewModel.cs]: rider did that

[the remark this answers is not in the document]
Comment (sofokli1) [reply Toggl.Core.UI/ViewModels/MainViewModel.cs]: do you want me to add empty spaces?

[the remark this answers is not in the document]
Comment (arturs105) [reply Toggl.Core.UI/ViewModels/MainViewModel.cs]: Naah

**Comment by sofokli1** [comment]
@juanlaube can you take a look at the last commit?

**Pull request merged**